Makefile

```diff
@@ -0,0 +1,43 @@
+TARGET=target
```

Comment:
A note with the expected results specified in the makefile and perhaps copyright stuff would be helpful

Reply:
Updated.